input/pagecontent/changes.md

```diff
@@ -113,6 +111,7 @@ This change log documents the significant updates and resolutions implemented fr
   - removed Must Support from Observation.encounter [FHIR-45134](https://jira.hl7.org/browse/FHIR-45134)
   - removed Must Support from Observation.performer [FHIR-44786](https://jira.hl7.org/browse/FHIR-44786) 
   - removed the fixed value constraint 'final' on Observation.status [FHIR-45120](https://jira.hl7.org/browse/FHIR-45120)
+  - added profile-specific implementation guidance on representing a specific pulse rate in the heart rate profile [FHIR-44861](https://jira.hl7.org/browse/FHIR-44861) 
```

Comment:
Change "added profile-specific implementation guidance on representing a specific pulse rate in the heart rate profile" to be "added profile-specific implementation guidance on representing pulse rate observations"

Reply:
Fixed.